lib/kitchen/transport/speedy_ssh.rb

```diff
@@ -28,12 +28,12 @@ class Connection < Ssh::Connection
         include SpeedyConnectionBase
 
         def valid_local_requirements?
-          !Mixlib::ShellOut.new("which tar > /dev/null").run_command.error?
+          !Mixlib::ShellOut.new("which tar && which gzip > /dev/null").run_command.error?
         end
 
         def valid_remote_requirements?
           begin
-            execute("which tar > /dev/null")
+            execute("which tar && which gzip > /dev/null")
```

Comment:
which tar > /dev/null && which gzip > /dev/null

(else only taken into account for last command, not sure there is another way to make it happen both in a single > )

Reply:
I've pushed a new commit to fix that, thanks. Using a subshell allows to do it with a single > - at the cost of spawning a shell, which is acceptable in my opinion :)